.ci/check_man.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+if [ $CIRCLECI ]; then
+    sudo apt install man
+    export MANPATH=$MANPATH:~/.local/man
```

Comment:
You've verified it produces a correct manpage which can be read by `man` (this might even be a good addition to the real test suite, for the man page building command), but you've also verified that it can *not* be found by `man`.

The problem with this is that `man coala` does not actually work by default for users.

We would have to also add documentation for users to add this line somewhere in the startup scripts, or add a different line depending on how they installed coala.

Before adding documentation, ideally we find a better way so that it 'just works'.

also c.f. https://gitlab.com/coala/mobans/issues/1#note_80986917

Reply:
So apparently we are using two space indents, but it isnt enforced yet.

Reply:
> We would have to also add documentation for users to add this line somewhere in the startup scripts, or add a different line depending on how they installed coala.

Actually, if virtualenv is not used then man pages are installed correctly. In case it is - then yes, users need to add directory in virtualenv to `MANPATH`. Do we need to create a separate issue for that? If so, I can work on adding required information to the docs.
Alternatively, we can think of a way to add required directory to `MANPATH` automatically, but I don't think it is a good idea.

> Before adding documentation, ideally we find a better way so that it 'just works'.

Unluckily, there is no way to specify absolute paths in `data_files`. So, if virtualenv is used, man files are placed relatively to virtualenv root.

> also c.f. https://gitlab.com/coala/mobans/issues/1#note_80986917

Yes, I am going to leave some short comment there as soon as Gitlab goes up again.

> So apparently we are using two space indents, but it isnt enforced yet.

Okay, I will change indentation now.

Reply:
> Actually, if virtualenv is not used then man pages are installed correctly.

We need a test which proves this ;-)

Please change the commit message to "Related to https://github.com/coala/coala/issues/5321" and we'll merge this as an interim measure, as it does show what happens with a ~/.local install and with a virtualenv install.  But we need a test of non-virtual-env, real install.

Until we have that verified by CI, this issue isnt completely resolved.

Reply:
> We need a test which proves this ;-)

Hm, I took a fresh look at this and noticed that I was wrong before. If we use `/usr/share/man/man1` as data files path, then man pages are installed correctly and no changes to `MANPATH` are needed. So if we use that, we can remove `export MANPATH=...` for CircleCI, as it does not use virtualenv. There is permission problem though. If we use `/usr/share/man/man1` then man page can be only installed if running as root. If not - there will be `Permission denied` error during `pip install`. To solve that we need to run the whole CI job as root (not a good solution) or to use hack like `sudo chown $(whoami) -R /usr/share` (which I don't like too) to make installation possible without root access.
As for Travis - as I said there is no way to go outside virtualenv. If we need to install man pages correctly there, we will need to change CI job / add a separate so it does not use virtualenv.
What are your thoughts on this?

> But we need a test of non-virtual-env, real install.

Sure, as I said this can be done on CircleCI, but a bit problematic on Travis.

> Have you looked at what they are doing in https://github.com/wkentaro/wstool/blob/68003a95c2bde7b26101a9621de814ff82a73b64/setup.py#L21 ?

Yes, I have. They use `/usr/share` prefix for man page installation. That prefix allows to install man pages to `/usr/share/man/man1`. However, virtualenv users still need to add path to `MANPATH`.